packages/commonwealth/client/scripts/views/components/LinksArray/types.ts

```diff
@@ -42,5 +42,5 @@ export type LinksArrayProps = {
 
 export type LinksArrayHookProps = {
   initialLinks: Link[];
-  linkValidation?: z.ZodEffects<z.ZodString, string, string> | z.ZodString;
+  linkValidation?: typeof snapshotValidationSchema;
```

Comment:
This will break the webhooks input array. I suggest we undo the last commit. I believe its not needed as we already have the snapshot validation to check for `http/s` logic